eng/pipelines/templates/steps/download-credscan-suppressions.yml

```diff
@@ -0,0 +1,12 @@
+# 1es PT credscan step will fail if the suppression file path we use in other stages does not exist here.
```

Comment:
I'm good with this for now but I really want to eliminate these templates. I know @hallipr is working on a strategy for the .NET release where we don't need this any longer.

Reply:
Yeah, @benbp commented when we had to add this that you weren't going to like it. I realize this will have to change both here and Java.

Reply:
If we can remove the build tools repo that will solve it, or are we trying another approach?